index.html

```diff
@@ -473,23 +473,23 @@
 					if (data.network.mode === "wifi") 
 					{
 						// wifi is the network mode from adopt payload - add the wifi icon
-						$('#root_network').parent().append('<div id="icons"><i class="mdi mdi-wifi" id="wifi-icon" aria-hidden="true" style="color:#2C448E;font-size: 30px;"></i></div>');
+						$('#root_network').parent().append('<div id="icons"><i class="mdi mdi-wifi" id="wifi-icon" aria-hidden="true" title="Device Using Wifi" style="color:#2C448E;font-size: 30px;"></i></div>');
```

Comment:
I think I would prefer to see two local variables defined, one for the icon name and one for the title. Then have a single line to add the `<div>` etc, substituting those variables.

For one it means all that complex HTML code is only defined once. 

Then I would just do `var title = data.network.mode`. So the tooltip is the raw network mode from the adoption payload.

Reply:
I'm not sure that's possible